### ISSUE TITLE
[Task]: Merge the two theme review queues into one

### ISSUE THREAD

**Issue by wagnerand** [mozilla/addons#14935]
### Description

We want to merge the "New" and "Updates" theme queues into a single queue.

The queue should still show the flags and due date as is, including sorting by due date.
One change that we want to make is rename the "Due Date" column on the queue page to "Target Date".

### Acceptance Criteria

```[tasklist]
  ### Milestones/checkpoints
  - [ ] There is only one single queue for themes and all queued themes are enqueued in that queue.
  - [ ] For that new queue only, the column with the due date should say "Target Date" instead.
```

### Checks

- [X] If the issue is ready to work on, I have removed the "needs:info" label.
- [X] If I have identified that the work is specific to a repository, I have removed "repository:addons-server" or "repository:addons-frontend"

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/AMOENG-954)

**Comment by diox** [comment]
To be nice to reviewers we should add redirects from the old URLs to the new merged one

**Comment by ioanarusiczki** [comment]
On AMO Dev:
- there's an Awaiting Review queue with Add-on, Flags and Target Date , oldest entries are the first in order and there's no sorting yet by the target date. 
- submitted themes in xpi format or using the theme wizard. Approved and submitted new versions. Added developer comments or reported and appealed on listed themes. I didn't see issues, all showed up in the queue. 
- [theme reviewers ](https://addons-internal.dev.mozaws.net/en-US/admin/models/access/group/50004/change/) have access to the new queue. I tried a few actions such as approve, reject or to deny an appeal (because of an abuse report) and it worked.

So the only question is about that sort if it requires a followup. 

Also checking other queues I've noticed a 500 server error on dev and stage when clicking Maliciousness Score or Metada Weight , Total Weight (in Manual Review Queue and Pending Rejection) but I think that's quite old.
